igvc_navigation/GraphSearch/include/Graph.h

```diff
@@ -100,6 +158,19 @@ class Graph
   // to maintain lower bounds in the priority queue
   float KeyModifier = 0;
 
+  float OccupancyThreshold = 0.7f;
```

Comment:
Magic Number.

Reply:
This is the default value but this value can be overridden with a launch parameter